roxie/ccd/ccdfile.cpp

```diff
@@ -37,6 +37,10 @@
 #ifdef __linux__
 #include <sys/mman.h>
 #endif
+#if defined (__linux__) || defined (__APPLE__)
```

Comment:
The code that calls syscall doesn't do anything if apple is defined - so it would be clearer without.